Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -104,12 +104,35 @@ protected virtual bool HasContentAttribute
       get { return false; }
     }
 
+    /// <summary>
+    /// Provides secure forwarding of additional attributes to  <see cref="BocColumnRenderingContext"/>
+    /// </summary>
+    public struct ParamObjectWithWriter
```

Comment:
I don't think `ParamObjectWithWriter` was intended as the final name @MichaelKetting? How about `AttributeRenderingContext` or something?

The struct should be moved to the top of the file as nested types belong at the top. We should get rid of the `secure` in the description as the struct has nothing to do with security.

Reply:
typical naming style etc:
`public readonly struct BocTitleCellRenderArguments`
`public readonly struct BocDataCellRenderArguments`
so: `BocDataCellAttributeRenderingContext` and  `BocTitleCellAttributeRenderingContext`?
not nested, we're  in a specific namespace anyway and the other types aren't nested either.